problem2/problem2_spec.rb

```diff
@@ -0,0 +1,8 @@
+require "./problem2"
+describe Problem2 do
+	it "Largest 2 Numbers" do
```

Comment:
How about test cases with an array of 1 element? or an array with that has the first and second max duplicated (Ex: [4, 4, 1, 3]) ?